### DISCUSSION TITLE
Deny School Name Resolution Question

### DISCUSSION THREAD

**Discussion by rldn12134** [community/community#60277] [GitHub Education]
### Hi everyone,

I have applied for GitHub Education several times. They were all rejected. I modified both the name of the profile and the picture of the document. Only the answer was that the school name was unknown.
Should I write the same spacing for the school name?
Also, school domain is not authenticated. I did it by regular mail.

**Comment by shivvamm** [comment]
Hello there,
I know sometimes it happens try using exact name of your educational institution as it appears on official documents or records. Use the same spacing and capitalization to avoid any confusion or discrepancies. If possible, try to use an authenticated school domain email address during the application process that really help a lot and it works most of the time.
In addition to these tips, it may be helpful to reach out to GitHub Support directly for more specific guidance. They can provide insights into the reasons behind the rejection and offer advice on how to improve your application.
